papis/commands/rm.py

```diff
@@ -1,5 +1,11 @@
 """
-Command-line Interface
+The command to remove Papis documents. Can also be used to remove files within
+documents, rather than an entire document.
+
+Prompts for confirmation before removing, showing information about what is
+being removed.
+
+Command-line interface
```

Comment:
This is the largest "addition" I've made to the docs within this PR. I thought this command's doc page looked a little jarring compared to the others without having a small intro (as all others do).

Reply:
I'll give it a look, but it's very welcome :heart: